pymongo/mongo_client.py

```diff
@@ -1583,6 +1583,8 @@ def _process_kill_cursors(self):
             try:
                 self._cleanup_cursor(True, cursor_id, address, sock_mgr,
                                      None, False)
+            except InvalidOperation as e:
+                raise e
             except Exception:
                 helpers._handle_exception()
```

Comment:
There's another try/except thats needs to be updated below too.

Reply:
Done.

Reply:
It's still missing (line 1602).

Reply:
Done.